rules/integrations/aws/defense_evasion_rds_instance_restored.toml

```diff
@@ -36,31 +33,36 @@ tags = [
     "Domain: Cloud",
     "Data Source: AWS",
     "Data Source: Amazon Web Services",
+    "Data Source: AWS RDS",
     "Use Case: Asset Visibility",
     "Tactic: Defense Evasion",
 ]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:rds.amazonaws.com and event.action:RestoreDBInstanceFromDBSnapshot and
-event.outcome:success
+any where event.dataset == "aws.cloudtrail" 
```

Comment:
No category exists for these events?

Reply:
When I've tried using the categories that do exist for aws I get failed unit tests, maybe because of the inconsistencies across Cloud and SaaS we've opted not to support them here? Not sure